examples/with-next-iframe/config/frontendConfig.js

```diff
@@ -13,6 +13,7 @@ export const frontendConfig = () => {
             EmailPassword.init(),
             Session.init({
                 isInIframe: true,
+                tokenTransferMethod: "header",
```

Comment:
does this make the examlpe work on safari and incognito chrome? If yes, please update the readme in this example app cause it has a "### IMPORTANT:" section which needs to be removed.